commercetools-models/src/main/java/io/sphere/sdk/categories/commands/updateactions/SetAssetDescription.java

```diff
@@ -0,0 +1,35 @@
+package io.sphere.sdk.categories.commands.updateactions;
+
+import io.sphere.sdk.categories.Category;
+import io.sphere.sdk.commands.UpdateActionImpl;
+import io.sphere.sdk.models.LocalizedString;
+
+import javax.annotation.Nullable;
+
+/**
+ * Changes the name of an asset.
```

Comment:
*"Changes the description"
By the way, could you fix this same mistake from the Product's action `SetAssetDescription`? ;)